scripts/deploy/src/install.js

```diff
@@ -0,0 +1,163 @@
+const fs = require('fs');
+const { execSync } = require('child_process');
+const axios = require('axios');
+const yaml = require('js-yaml');
+const path = require('path');
+
+const MEDIC_REPO_NAME = "medic";
+const MEDIC_REPO_URL = "https://docs.communityhealthtoolkit.org/helm-charts";
+const CHT_CHART_NAME = `${MEDIC_REPO_NAME}/cht-chart-4x`;
+const DEFAULT_CHART_VERSION = "0.2.3";
```

Comment:
Set to wildcard
```suggestion
const DEFAULT_CHART_VERSION = "1.0.*";
```
(or `1.*.*`) you might also need to quote the usage of the version to ensure it doesn't expand the wildcard